docs/deployment/configuring_logging.md

```diff
@@ -0,0 +1,67 @@
+# Configuring the Logging stack via Gardenlet configurations
+
+# Enable the Logging
+
+Logging feature gate must be enabled in order to install the Gardener Logging stack.
+
+In the Gardenlet configuration add:
+```yaml
+featureGates:
+  Logging: true
+```
+
+From now on each Seed is going to have a logging stack which will collect logs from all pods and some systemd services. Logs related to Shoots with `testing` purpose are dropped in the `fluent-bit` output plugin. Shoots with different than `testing` purpose have the same type of log aggregator(but different instance) as the Seed. The logs can be viewed in the Grafana in the `garden` namespace for the Seed components and in the  `shoot controlplane` namespace for each Shoot.
+
+# Enable logs from the Shoot's node systemd services.
+
+The logs from the systemd services on each node can be retrieved by enabling the `logging.shootNodeLogging` option.
+
+In the Gardenlet configuration:
+```yaml
+featureGates:
+  Logging: true
+logging:
+  shootNodeLogging:
+    shootPurposes:
+    - "evaluation"
+    - "deployment"
+```
+
+Under the `shootPurpose` section just list all the shoot purposes for which the Shoot node logging feature will be enabled. `Testing` purpose is redundant becaus this purpose prevent the logging stack installation.
```

Comment:
```suggestion
Under the `shootPurpose` section just list all the shoot purposes for which the Shoot node logging feature will be enabled. Specifying the `testing` purpose has no effect because this purpose prevents the logging stack installation.
```

or?